README.md

```diff
@@ -3,15 +3,11 @@ OpenRA Web
 
 Uses the [nanoc](http://nanoc.ws/) static website generator. [![Build Status](https://travis-ci.org/OpenRA/OpenRAWeb.png?branch=master)](https://travis-ci.org/OpenRA/OpenRAWeb)
 
-Deployment
+Development
 ----------
-* `nanoc compile` to update the HTML.
-* `nanoc view` to preview the changes at [localhost:3000](http://localhost:3000).
-* `nanoc deploy --target res0l` to upload to [open-ra.org](http://openra.res0l.net) (requires SSH access and rsync).
-* Push to [openra.github.io](https://github.com/OpenRA/openra.github.io.git) to update [openra.net](http://openra.net) (requires git)
+Use `nanoc compile` and `nanoc view` to generate and host a static preview at [localhost:3000](http://localhost:3000), or `nanoc autocompile` to automatically recompile as required.  Changes will be automatically deployed to [openra.github.io](https://github.com/OpenRA/openra.github.io.git) when a pull request is merged.
```

Comment:
Indeed, but we are stuck on an outdated version of nanoc where the command works fine, while their recommended strategy requires pulling in a bunch more ruby crap.